include/Core/DB/TableManager.php

```diff
@@ -315,9 +315,9 @@ public function fetch_log_item_by_item_data( $data ) {
 		// Execute the following `if` conditions only when $data is array.
 		if ( array_key_exists( 'to', $data ) ) {
 			// Since the value is stored as CSV in DB, convert the values from error data to CSV to compare.
-			$data['to'] = Util\join_array_elements_with_delimiter( $data['to'] );
+			$to_email = Util\join_array_elements_with_delimiter( $data['to'] );
 
-			$to_email = trim( esc_sql( $data['to'] ) );
+			$to_email = trim( esc_sql( $to_email ) );
```

Comment:
Note from Scrutinizer

`It seems like esc_sql($to_email) can also be of type array; however, parameter $str of trim() does only seem to accept string, maybe add an additional type check?`

Reply:
@sudar Though the note has already been handled, yet we need to figure out a way to suppress  the message.

This line https://github.com/sudar/email-log/blob/171-fix/scrutinizer-issue/include/Core/DB/TableManager.php#L318 will always return a string and not an array.